torch/distributed/_composable/fsdp/_fsdp_param.py

```diff
@@ -28,23 +28,25 @@
   on the module when applying FSDP
 - Sharded parameter: sharding the original parameter on dim-0 as a DTensor
   over the main mesh
-- All-gather input: the ``torch.Tensor`` passed to all-gather, derived from the
-  sharded parameter
-- All-gather output: the ``torch.Tensor`` resulting from all-gathering the
-  all-gather input
+- All-gather inputs: the ``torch.Tensor`` or ``Tensor`` s passed to all-gather,
```

Comment:
shall we explicitly say Tensor subclass? ``torch.Tensor`` or ``Tensor`` seems identical

Reply:
Oh I meant like it can be 1 or more `torch.Tensor`s.

There is an s after the \`\`Tensor\`\`. I put a space in between because that is usually needed when rendering public docs, so it was a habit (to explain why I wrote  \`\`Tensor\`\`s but rather \`\`Tensor\`\` s).